qid-batch-runner.yml

```diff
@@ -101,8 +106,10 @@ spec:
             value: "/home/qidbatchrunner/.sftp-ssh/id_rsa"
           - name: OUR_PUBLIC_KEY_PATH
             value: "/home/qidbatchrunner/.pgp-keys/our_public_key.asc"
-          - name: OTHER_PUBLIC_KEY_PATH
-            value: "/home/qidbatchrunner/.pgp-keys/other_public_key.asc"
+          - name: QM_PUBLIC_KEY_PATH
+            value: "/home/qidbatchrunner/.pgp-keys/supplier_QM_dummy_public.asc"
+          - name: PPO_PUBLIC_KEY_PATH
+            value: "/home/qidbatchrunner/.pgp-keys/supplier_ppo_dummy_public.asc"
```

Comment:
would be nice if these keys were consistently named

Reply:
`dummy_ppo_supplier_public_key.asc` should be renamed to `supplier_PPO_dummy_private.asc`